app/views/trails/show.json.jbuilder

```diff
@@ -8,4 +8,11 @@ json.distance @trail.distance
 json.duration @trail.duration
 json.food_supply @trail.food_supply
 json.warning @trail.warning
-json.photo_urls @trail.photo_urls
\ No newline at end of file
+json.photo_urls @trail.photo_urls
+json.start_longitude @trail.start_longitude
+json.start_latitude @trail.start_latitude
+json.end_longitude @trail.end_longitude
+json.end_latitude @trail.end_latitude
+json.region_id @trail.region_id
+json.birds @trail.region.birds
```

Comment:
if we do this we can remove the show.json.jbuilder + index.json.jbuilder on plants and birds views.  also their controllers are not used anymore.  I can fix after merge